redaxo/src/core/boot.php

```diff
@@ -139,6 +139,8 @@
     }
 }
 
+rex_extension::register('SESSION_REGENERATED', rex_backend_login::sessionRegenerated(...));
```

Comment:
könnte diese syntax zu nem parse error führen, wenn man mit zu alter php version das setup starten will?

Reply:
Hmm guter Punkt. Wir brechen bei falsche PHP-Version ja direkt oben in der boot.php ab.
Aber wir sind hier ja genau in der boot.php, daher sollten wir in der boot.php vermutlich keine neuere Syntax verwenden. Ich checke das nochmal, wie es sich verhält.

Reply:
anders herum gesagt: vllt können wir die "paar dateien" die notwendig sind um im setup mit veralteter php version bis zum "gewünschten error" zu gelangen mittels github action in einer alten php version linten (oder ggf. das setup einfach nur starten lassen und schauen ob der gewünschte error kommt)

Reply:
Bis zum Setup kommt man nicht. Das haben wir schon sehr lange aufgegeben. 
Finde ich auch nicht wichtig. 

Aber bis zur Meldung in der boot.php (die wir zuletzt ja hier optimiert haben: https://github.com/redaxo/redaxo/pull/5534) sollte man halt kommen. Und dafür könnte man in der Tat mal eine Github-Action einrichten, die das sicherstellt.

Reply:
> Aber bis zur Meldung in der boot.php (die wir zuletzt ja hier optimiert haben: https://github.com/redaxo/redaxo/pull/5534) sollte man halt kommen. Und dafür könnte man in der Tat mal eine Github-Action einrichten, die das sicherstellt.

das meinte ich